homeassistant/components/device_tracker/luciwifi.py

```diff
@@ -1,224 +0,0 @@
-"""
-Support for OpenWRT (luci) routers.
-
-For more details about this platform, please refer to the documentation at
-https://home-assistant.io/components/device_tracker.luciwifi/
```

Comment:
The URL is not working.
Should be _https://www.home-assistant.io/components/device_tracker.luci/_  ?

Reply:
I believe this would be a new page then

Reply:
we should probably add/rename the page to openwrt as this is what the device it will track. But it won't be worth it if this doesnot gets merged.